main/index.js

```diff
@@ -97,32 +95,14 @@ app.on('second-instance', () => {
   ctx.showUI()
 })
 
-if (isDev) {
-  // Do not preserve old Activity entries in development mode
-  (async () => {
-    try {
-      await fs.writeFile(await getActivityFilePath(), '')
-    } catch {}
-  })()
-}
```

Comment:
Could you please remind me what is the new behaviour? IIUC, we agreed that Activity entries are no longer persisted, is that correct? If that's the case, then we don't need special handling for the development mode, and this removal is fine 👍🏻

Reply:
This is correct 👍